pkg/cmd/attestation/verification/sigstore.go

```diff
@@ -51,108 +55,98 @@ var ErrNoAttestationsVerified = errors.New("no attestations were verified")
 // Public Good, GitHub, or a custom trusted root.
 func NewLiveSigstoreVerifier(config SigstoreConfig) *LiveSigstoreVerifier {
 	return &LiveSigstoreVerifier{
-		config: config,
+		TrustedRoot:  config.TrustedRoot,
+		Logger:       config.Logger,
+		NoPublicGood: config.NoPublicGood,
+		TrustDomain:  config.TrustDomain,
 	}
 }
 
-func (v *LiveSigstoreVerifier) chooseVerifier(b *bundle.Bundle) (*verify.SignedEntityVerifier, string, error) {
+func getBundleIssuer(b *bundle.Bundle) (string, error) {
```

Comment:
This logic was originally inside of the `chooseVerifier` method. I moved it out so we could parse the bundle's issuer separately. This allowed me to clean up `chooseVerifier` a bit.